cip-new/cip-new.md

```diff
@@ -0,0 +1,70 @@
+## CIP XXXX
+
+<pre>
+ CIP: CIP XXXX
+ Title: Bootstrap splice node from non-zero round
+ Author: Julien Tinguely
+ Comments-Summary: No comments yet
+ Status: Draft
+ Type: Standards Track
+ Created: 2025-07-09
+ Approved: ?
+ License: CC0-1.0
+</pre>
+
+## Abstract
+
+This CIP proposes to modify the network's bootstrapping process allowing SV nodes to start from a pre-configured non-zero round. 
+Currently, on network resets, all SV nodes begin at round zero, which also means they revert to the initial coin issuance curve. 
+Our goal is to enable network resets that maintain historical round progression, rather than resetting to zero. 
+This approach aims to better align with MainNet's ongoing state.
+
+## Specification
+
+### Daml
+
+A new optional argument `initialRound` is added to the `DsoRules_Bootstrap` choice. 
+When provided, this value is used as the initial round in the network and changes the issuance periods accordingly.
+
+### SV Application
+
+The SV Application is modified so that it reads the new optional flag `SPLICE_APP_SV_INITIAL_ROUND` defaulted to 0.
+Each SV Application compares its value to its onboarding sponsor's value before to save it to the user metadata to ensure
+all SV nodes use the same initial round.
```

Comment:
```suggestion
The SV Application is modified so that it reads the new optional flag `SPLICE_APP_SV_INITIAL_ROUND` defaulted to 0.
Each SV Application compares its value to its onboarding sponsor's value when joining after a reset. This ensures that a misconfiguration by either the sponsor or the joining node is caught.
```